docs/source/glossary.rst

```diff
@@ -39,7 +39,7 @@ Glossary
       A title for a classification/categorization of data according to how it is intended to be shared.
 
     MessageKit
-      The :term:`KEM` and :term:`DEM` encrypted data that is stored together.
+      The :term:`KEM` and :term:`DEM` encrypted data that are stored together.
```

Comment:
Something that was missing from a previous PR is to remove references to KEM and DEM. I think it's not NuCypher specific, and additionally, it's not being used 100% correct. I think we can safely define MessageKits this in terms of Capsule and Ciphertext, as Umbral does.

Reply:
@KPrasch added a comment to the previous PR to indicate that it made sense to define KEM/DEM usage in NuCypher's specific case - https://github.com/nucypher/nucypher/pull/1377#discussion_r330188704.

Updated the definition to use `Capsule` and `ciphertext`

Reply:
✔️